Tasks/AzureResourceManagerTemplateDeploymentV3/task.json

```diff
@@ -0,0 +1,297 @@
+{
+    "id": "94A74903-F93F-4075-884F-DC11F34058B4",
+    "name": "AzureResourceManagerTemplateDeployment",
+    "friendlyName": "ARM template deployment",
+    "description": "Deploy an Azure Resource Manager (ARM) template to all the deployment scopes",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/deploy/azure-resource-group-deployment",
+    "helpMarkDown": "[Learn more about this task](https://aka.ms/armdeploymenttaskreadme)",
+    "category": "Deploy",
+    "releaseNotes": "- Added support for deployment at all the deployment scopes.\n- Removed all the VM related actions.",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 3,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "demands": [],
+    "minimumAgentVersion": "2.119.1",
+    "groups": [
+        {
+            "name": "AzureDetails",
+            "displayName": "Azure Details",
+            "isExpanded": true
+        },
+        {
+            "name": "Template",
+            "displayName": "Template",
+            "isExpanded": true,
+            "visibleRule": "action = Create Or Update Resource Group || deploymentScope != Resource Group"
+        },
+        {
+            "name": "Advanced",
+            "displayName": "Advanced",
+            "isExpanded": true,
+            "visibleRule": "action = Create Or Update Resource Group || deploymentScope != Resource Group"
+        }
+    ],
+    "inputs": [
+        {
+            "name": "deploymentScope",
+            "type": "pickList",
+            "label": "Deployment Scope",
```

Comment:
Provide a hyperlink to go to Deployment Scopes Azure docs

Reply:
Done